File: app/src/main/java/com/compuspulse/utils/HtmlUtils.java
package com.compuspulse.utils;

import androidx.core.text.HtmlCompat;

public class HtmlUtils {

    private HtmlUtils() {
    }

    public static String plainText(String value) {
        if (value == null) {
            return "";
        }
        return HtmlCompat.fromHtml(value, HtmlCompat.FROM_HTML_MODE_LEGACY)
                .toString()
                .trim();
    }
}

File: app/src/main/java/com/compuspulse/utils/HtmlUtils.kt
package com.compuspulse.utils

import androidx.core.text.HtmlCompat

object HtmlUtils {

    @JvmStatic
    fun plainText(value: String?): String {
        if (value == null) {
            return ""
        }
        return HtmlCompat.fromHtml(value, HtmlCompat.FROM_HTML_MODE_LEGACY)
            .toString()
            .trim()
    }
}
